src/user/screens/starred-repository-list.screen.js

```diff
@@ -52,7 +89,18 @@ class StarredRepositoryList extends Component {
           )}
         {!isPendingStarredRepositories && (
           <FlatList
-            data={starredRepositories}
+            data={starredRepos}
+            onRefresh={() =>
+              this.props.getStarredReposForUser(userId, {
+                forceRefresh: true,
+              })
+            }
```

Comment:
Remember how our action set contains the 3 usual pending,success,error and an addition 'reset' thingy?
'reset' is dispatched when 'forceRefresh' is set to `true`, and it will simply wipe out  `STORE.pagination.ACTIVITY_GET_STARRED_REPOS_FOR_USER['machour']`, so that fresh information is refetched from the GitHub endpoint

Reply:
Was just about to ask about why we have `reset`. You've thought this through mate :)